gemfiles/rails-latest-release.gemfile

```diff
@@ -2,4 +2,4 @@ source "https://rubygems.org"
 
 gemspec path: '..'
 
-gem 'rails', '~> 5.1.0'
+gem 'rails'
```

Comment:
@rafaelfranca I wanna confirm it makes sense to test against rails since the goal is to see how the gem will work against a rails application. And that this is better than testing against railties versions only

Reply:
👍 It does make sense.